botocore/parsers.py

```diff
@@ -536,6 +536,17 @@ def _handle_float(self, shape, text):
     def _handle_timestamp(self, shape, text):
         return self._timestamp_parser(text)
 
+    @_text_content
+    def _handle_expires_timestamp(self, shape, text):
```

Comment:
Why are we adding this to the `BaseXMLResponseParser` but invoking it in the `BaseRestParser`?

Reply:
I will move it to `BaseRestParser`.